controllers/admin/AdminRequestSqlController.php

```diff
@@ -50,10 +50,7 @@ public function __construct()
         $this->fields_list = array(
             'id_request_sql' => array('title' => $this->trans('ID', array(), 'Admin.Global'), 'class' => 'fixed-width-xs'),
             'name' => array('title' => $this->trans('SQL query Name', array(), 'Admin.Advparameters.Feature')),
-            'sql' => array(
-                'title' => $this->trans('SQL query', array(), 'Admin.Advparameters.Feature'),
-                'filter_key' => 'a!sql',
-            ),
+            'sql' => array('title' => $this->trans('SQL query', array(), 'Admin.Advparameters.Feature')),
```

Comment:
I think that with the other modification in AdminController, this one is no longer useful